app/models/commConns.js

```diff
@@ -19,6 +33,35 @@ class CommConns {
       }).catch(reject);
     })
   }
+
+  static findByCommId (communicationId) {
```

Comment:
Went ahead and updated such that:

```
  static findManyByAttribute(attributeName, value, otherOperations) {
    if (!otherOperations) {
      otherOperations = (dbCall) => {
        return dbCall;
      }
    }
    this._checkModelValidity();
    return new Promise((fulfill, reject) => {
      let basicDbCall = db(this.tableName)
                          .where(attributeName, value);
      otherOperations(basicDbCall)
      .then((objects) => {
        return this._getMultiResponse(objects, fulfill)
      }).catch(reject)
    })
  }
```

Is in models and it looks like this when calling from CommConns:

```
      CommConns.findManyByAttribute("comm", 1, (dbCall) => {
        return dbCall
          .andWhere("retired", null);
      })
      .then((commConns) => {
        clients.length.should.be.exactly(commConns.length);
        done();
      }).catch(done);```